frontend/src/views/ComplianceReports/components/_schema.jsx

```diff
@@ -9,11 +9,11 @@ export const reportsColDefs = (t, bceidRole) => [
     field: 'compliancePeriod',
     headerName: t('report:reportColLabels.compliancePeriod'),
     width: 210,
-    cellRenderer: LinkRenderer,
-    cellRendererParams: {
-      url: ({ data }) => `${data.compliancePeriod?.description}/${data.complianceReportId}`,
-    },
-    valueGetter: ({ data }) => data.compliancePeriod?.description || ''
+    suppressFloatingFilterButton: true,
```

Comment:
Was it intentional to remove the cellRenderer: LinkRenderer here? Will that make the columns look different than before?

Reply:
You are making me wonder if I should get rid of the other cellRenderers in organization and status. What is the purpose of this if it is not to allow the user to click on the row and take the user to the details? This is done in ComplianceReports.jsx:
```
const handleRowClicked = useCallback(
    ({ data }) => {
      navigate(
        ROUTES.REPORTS_VIEW.replace(
          ':compliancePeriod',
          data.compliancePeriod.description
        ).replace(':complianceReportId', data.complianceReportId)
      )
    },
    [navigate]
  )
```

Reply:
I'm not exactly sure to be honest, might be a better question for @AlexZorkin or maybe its safe to delete? I imagined it was to make it look like a blue link.